src/plugins/telemetry_management_section/public/components/telemetry_management_section.test.tsx

```diff
@@ -29,6 +29,7 @@ describe('TelemetryManagementSectionComponent', () => {
 
   it('renders as expected', () => {
     const onQueryMatchChange = jest.fn();
+    const isSecurityExampleEnabled = jest.fn().mockReturnValue(true);
```

Comment:
Just a nit, but might be more DRY to just set these (`isSecurityExampleEnabled` && `onQueryMatchChange`) in a `beforeEach` at the top of the relevant describe blocks since they don't seem to change for any of the tests

Reply:
I usually prefer to keep them like that in the tests to make it easier to read what is happening. each `it` has almost all it needs encapsulated inside it, so you can update/refactor the tests more easily if needed in the future.